examples/1-hello-world/README.md

```diff
@@ -135,9 +132,7 @@ with the following arguments:
 }:
 ```
 
-As shown above, we declare this function with three arguments: `system`,
```

Comment:
Exactly.  Reader is going to see the arguments enumerated right below here, so no need to list twice.

Reply:
Yup, exactly what I was thinking when I read it over again. 😁